services/subscriber/udp.go

```diff
@@ -0,0 +1,41 @@
+package subscriber
+
+import (
+	"net"
+
+	"github.com/influxdb/influxdb/cluster"
+)
+
+// Writes points over UDP using the line protocol
+type UDP struct {
+	addr string
+}
+
+func NewUDP(addr string) *UDP {
```

Comment:
Can we add stats to this service? Full details on how to do this are here:

https://github.com/influxdb/influxdb/blob/master/monitor/README.md#registering-statistics-and-diagnostics

It's very easy to do.